lib/css/base/_stacks-configuration-dynamic.less

```diff
@@ -21,38 +21,9 @@
     @body-font-color:                                var(--theme-body-font-color);
     @body-type-face:                                 var(--theme-body-font-family);
     @body-background:                                var(--white);
-    @body-site-width:                                1060px;
-
-    //  Links
-    @link-color:                                     var(--theme-secondary-400);
-    @link-color-hover:                               var(--theme-secondary-350);
-    @link-color-visited:                             var(--theme-secondary-500);
-
-    //  Button Default
-    @button-color:                                   var(--theme-secondary-400);
-    @button-background-color:                        transparent;
-    @button-hover-color:                             var(--theme-secondary-500);
-    @button-hover-background-color:                  var(--theme-secondary-050);
-    @button-active-background-color:                 var(--theme-secondary-100);
-    @button-selected-color:                          var(--theme-secondary-900);
-    @button-selected-background-color:               var(--theme-secondary-150);
-
-    //  Button Filled
-    @button-filled-color:                            var(--theme-secondary-700);
-    @button-filled-background-color:                 var(--theme-secondary-050);
-    @button-filled-border-color:                     var(--theme-secondary-350);
-    @button-filled-hover-color:                      var(--theme-secondary-800);
-    @button-filled-hover-background-color:           var(--theme-secondary-100);
-    @button-filled-active-background-color:          var(--theme-secondary-150);
-    @button-filled-active-border-color:              var(--theme-secondary-350);
-    @button-filled-selected-color:                   var(--theme-secondary-900);
-    @button-filled-selected-background-color:        var(--theme-secondary-300);
-    @button-filled-selected-border-color:            var(--theme-secondary-500);
-
-    //  Button Outline
-    @button-outlined-border-color:                   var(--theme-secondary-300);
-    @button-outlined-selected-border-color:          var(--theme-secondary-400);
+    @body-site-width:                                1060px; // TODO not overridden in prod
 
+    // TODO not overridden in prod
```

Comment:
Glad all these TODOs are identified. I wonder if these should actually be `// Deprecated` comments? Or, honestly, I'd be happy to just yeet these straight away.

Reply:
Changed to `// DEPRECATED` comments. I'm going to come back in a follow up PR to yeet these entirely once I make triple sure these are not used anywhere (I'm 90% sure they aren't though).